aten/src/ATen/BatchingRegistrations.cpp

```diff
@@ -1098,7 +1098,7 @@ TORCH_LIBRARY_IMPL(aten, Batched, m) {
   m.impl("movedim.int", static_cast<Tensor(*)(const Tensor&,int64_t,int64_t)>(native::movedim)); // composite wrt autograd
   // NB: static_cast because there's another variant of narrow. However, we don't
   // want to support the other variant yet bc it isn't documented...
-  m.impl("narrow", static_cast<Tensor(*)(const Tensor&,int64_t,int64_t,int64_t)>(native::narrow)); // composite wrt autograd
+  m.impl("narrow", static_cast<Tensor(*)(const Tensor&,int64_t,SymInt,SymInt)>(native::narrow_symint)); // composite wrt autograd
```

Comment:
Feels like the cast may not be necessary anymore

Reply:
i think you're right.